### PR TITLE
Mathmatics operations

### PR DESCRIPTION
**Pull request by Nizar7zak**

### CODE REVIEW

**Comment by AyahSidu** [comment]
Good job

File: mathmatics.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+a = 1
+b = 2
```
Comment (Nizar7zak) [inline mathmatics.py:2]: could you please check this value

File: mathmatics.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+c = 3
+d = 4
+e = 5
```
Comment (Mashharawi) [inline mathmatics.py:5]: May you add 70 here

File: mathmatics.py (hunk 1 above)
Comment (AyahSidu) [reply mathmatics.py:2]: it's good

Comment (AyahSidu) [reply mathmatics.py:2]: Good job

**Review by suhail-alhassanat — APPROVED** [review]

**Comment by Mashharawi** [comment]
where are y

File: mathmatics.py (hunk 2 above)
Comment (Nizar7zak) [reply mathmatics.py:5]: I did it, please check it now

**Review by suhail-alhassanat — APPROVED** [review]

**Pull request merged**